src/main/java/Luke.java

```diff
@@ -0,0 +1,237 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Luke {
+    /** An object to manage the responses of LUKE */
+    private static Response response;
+
+    /** A scanner object to read in user input */
+    private static Scanner scanner;
+
+    /** An object used to manage the tasks added by the user */
+    private static TaskOrganizer taskOrganizer;
+
+    /** A list of valid commands */
+    private static ArrayList<String> commands;
+
+    /** Initializes all the objects used in LUKE, says "Hi" to the user */
+    private static void initialize() {
```

Comment:
Perhaps in the function name can add a noun after initialize to know what are we initializing